opencga-app/app/scripts/azure/arm/solr/azuredeploy.json

```diff
@@ -280,5 +294,15 @@
         }
       }
     }
-  ]
+  ],
+  "outputs" : {
```

Comment:
Are we happy connecting over the private network unauthenticated for the time being?

Reply:
I'd be happier to use auth but I don't have any knowledge on how the `solr` handles authentication. Should we proceed with this for now to complete the integration testing and track an issue to resolve in the future?